readme.md

```diff
@@ -316,6 +325,17 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [linux/kernel/seccomp.c](https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/kernel/seccomp.c)
 - [linux/net/core/dev.c](https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/net/core/dev.c) - contains the function `dev_change_xdp_fd()` that is called through a Netlink command to hook a XDP program to a device, after is has been loaded into the kernel from user space. This function in turns uses a callback from the relevant driver.
 
+## eBPF in Security
+
+- [eBPF for security: a beginner's guide](https://redcanary.com/blog/ebpf-for-security) - Introduction on eBPF and how Red Canary uses it.
+- [Embrace The Red: Offensive BPF!](https://embracethered.com/blog/tags/ebpf) - A series of posts around BPF usage in offensive settings, and also how its misuse can be detected.
+- [Offensive BPF! Getting started](https://embracethered.com/blog/posts/2021/offensive-bpf/) - eBPF’s offensive usage is also slowly getting more attention. So, I decided to dive into the topic from a red teaming point of view to learn about it to raise awareness and share the journey.
+- [eBPF: Block Linux Fileless Payload "Malware" Execution with BPF LSM](https://djalal.opendz.org/post/ebpf-block-linux-fileless-payload-execution-with-bpf-lsm/)
+- eBPF Rootkit
+  - [Blackhat 2021: With Friends Like eBPF, Who Needs Enemies?](https://www.blackhat.com/us-21/briefings/schedule/#with-friends-like-ebpf-who-needs-enemies-23619) - Talk about an eBPF rootkit.
+  - [eBPF, I thought we were friends !](https://defcon.org/html/defcon-29/dc-29-speakers.html#fournier) - Similar talk as at Blackhat 2021, talk about an eBPF rootkit. 
+- [ebpfkit](https://github.com/Gui774ume/ebpfkit) - ebpfkit is a rootkit that leverages multiple eBPF features to implement offensive security techniques.
```

Comment:
```suggestion
- [ebpfkit](https://github.com/Gui774ume/ebpfkit) - A rootkit that leverages multiple eBPF features to implement offensive security techniques.
```
This item can be indented to appear in the “eBPF rootkit” list, I think this is the same rootkit as described in the two links above.

Reply:
removed the rootkit section, because also the blog posts on embrace the red talks about rookit functionality.